<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RectTestDialog</class>
 <widget class="QDialog" name="RectTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rectangle Tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="2">
          <widget class="QSlider" name="heightInput">
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSlider" name="xInput">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="xLabel">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSlider" name="yInput">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="heightLabel">
           <property name="text">
            <string>height</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="yLabel">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSlider" name="widthInput">
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="axis3dXLabel">
           <property name="text">
            <string>3d x</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="colorLabel">
           <property name="text">
            <string>color</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QSlider" name="axis3dYInput">
           <property name="minimum">
            <number>-199</number>
           </property>
           <property name="maximum">
            <number>199</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="axis3dYLabel">
           <property name="text">
            <string>3d y</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QSlider" name="axis3dXInput">
           <property name="minimum">
            <number>-199</number>
           </property>
           <property name="maximum">
            <number>199</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="angleLabel">
           <property name="text">
            <string>angle</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="colorInput"/>
         </item>
         <item row="6" column="2">
          <widget class="QSlider" name="angleInput">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="axis3dZLabel">
           <property name="text">
            <string>3d z</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QSlider" name="axis3dZInput">
           <property name="minimum">
            <number>-199</number>
           </property>
           <property name="maximum">
            <number>199</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="xInput2">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="yInput2">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="widthInput2">
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="heightInput2">
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QSpinBox" name="angleInput2">
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QDoubleSpinBox" name="axis3dXInput2">
           <property name="minimum">
            <double>-1.990000000000000</double>
           </property>
           <property name="maximum">
            <double>1.990000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QDoubleSpinBox" name="axis3dYInput2">
           <property name="minimum">
            <double>-1.990000000000000</double>
           </property>
           <property name="maximum">
            <double>1.990000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QDoubleSpinBox" name="axis3dZInput2">
           <property name="minimum">
            <double>-1.990000000000000</double>
           </property>
           <property name="maximum">
            <double>1.990000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="fileInput"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="fileLabel">
           <property name="text">
            <string>Datei</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="fileInput2">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDeclarativeView" name="qmlView">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RectTestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RectTestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xInput</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xInput2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xInput2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xInput</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yInput</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yInput2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yInput2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yInput</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthInput</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widthInput2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthInput2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widthInput</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleInput</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleInput2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleInput2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleInput</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightInput</sender>
   <signal>valueChanged(int)</signal>
   <receiver>heightInput2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightInput2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>heightInput</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
